views/play.erb

```diff
@@ -0,0 +1,14 @@
+
+<html>
+  <div>
+  
+    <h1> Ok <%= @name1 %> let's play Rock, Paper, Scissors!</h1>
+    <br>
+    <form action="/show_results" method="post">
+      <label for="name"> Make your choice! </label><br>
```

Comment:
Thanks for showing me how I can use the three buttons to make a choice! In my code I requested to type in the answer and your code makes it easier for the user